openshift/sandboxed-containers-operator
# Fix stale doc links for 1.5.3

**- Description of the problem which is fixed/What is the use case**

OSC documentation has moved out of OCP. Fix the links in the repo.

Fixes: [KATA-2523](https://issues.redhat.com//browse/KATA-2523)

**- What I did**

Fixed the links everywhere

**- How to verify it**

- Checkout the PR
- `git grep docs.openshift.com` is empty
- Verify links in `git grep /access.redhat.com` are valid
